### DISCUSSION TITLE
Can't invite a User

### DISCUSSION THREAD

**Discussion by jeremy-allocate** [Budibase/budibase#9900] [Help] [answered]
My org is on the free plan, and when I go to add a 6th user to our account it keeps erroring out and just says "unavailable" for that user after I submit to send the invite email (the password based flow doesn't work either, and neither does importing).

Has anyone ever seen this before?  It seems to be specific to this one user's email address, as I was just able to invite a different user with a different email and had no issue.

![image](https://user-images.githubusercontent.com/97991821/223244537-a490c69e-9865-4f49-9ba3-c89c279be40d.png)

**Comment by ConorWebb96** [comment, marked as answer]
Hey @jeremy-allocate,

From my knowledge this Unavailable reason can occur for a number of different reasons:
- The user is in another tenant.
- The email may already be assigned as the root user.

Personally I think that the issue lies with the email being already attached to another cloud tenant. Currently with Budibase cloud a user can only be on one tenant at a time.

A workaround for this would be to use the prefix +1 e.g. test+1@test.com. This work around will only be able to be done if your email provider supports it for example gmail is one which would support this feature.

**Reply by jeremy-allocate** [reply]
So strange, there's really no way/reason this user's email address should be in another tenant (it's a company email address and we only have one tenant)... but your +1 workaround worked for us so thanks for the tip & quick reply Conor!!
